jetty-http/src/main/java/org/eclipse/jetty/http/QuotedQualityCSV.java

```diff
@@ -18,13 +18,13 @@
 
 package org.eclipse.jetty.http;
 
-import static java.lang.Integer.MIN_VALUE;
-
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
 import java.util.function.Function;
 
+import static java.lang.Integer.MIN_VALUE;
```

Comment:
Is this used?

Reply:
Yup, looks like it's from the `secondaryOrdering` function (unique constructor) and the `sort()` method.

https://github.com/eclipse/jetty.project/blob/336ef0292591353e793ec4d1e5e723033fb6713b/jetty-http/src/main/java/org/eclipse/jetty/http/QuotedQualityCSV.java#L75-L87